linkerd/docs/protocol-h2.md

```diff
@@ -273,6 +273,7 @@ Key  | Default Value | Description
 ---- | ------------- | -----------
 namespace | (all) | The Kubernetes namespace where the ingress resources are deployed. If not specified, Linkerd will watch all namespaces.
 ingressClassAnnotation | `linkerd` | When using [multiple ingress controllers](https://github.com/kubernetes/ingress/blob/master/docs/faq/README.md#how-do-i-run-multiple-ingress-controllers-in-the-same-cluster), Linkerd will only use the ingress resource annotated with this class.
+strict | `false` | Ignore default backends - identify requests only when ingress rules exist.
```

Comment:
What do you think about calling this parameter "ignoreDefaultBackends" or something that's more self-explanatory than "strict"?

Reply:
And changing the description to "Identify requests only when _explicit_ ingress rules exist."

Reply:
Sounds good to me on both counts. Will change.